control-plane/connect-inject/constants/constants.go

```diff
@@ -54,3 +54,33 @@ const (
 
 	KubernetesSuccessReasonMsg = "Kubernetes health checks passing"
 )
+
+// GetDefaultConsulNamespace returns the default namespace if the passed namespace
+// is empty, otherwise returns back the passed in namespace.
+func GetDefaultConsulNamespace(ns string) string {
```

Comment:
~ I feel like the names for these are a bit confusing. WDYT about something like `GetNormalizedConsulNamespace`?

The name implies that we're getting the default, but that's only true when `ns` is empty.

Reply:
yeah, that sounds good